src/components/store-deploy-block.tsx

```diff
@@ -1,13 +1,15 @@
-import React, { Dispatch, FunctionComponent, useState } from "react";
-import { TextInput } from "./common/text-input";
+declare let window: any;
```

Comment:
1. dont put ts definitions in ts code
2. you completely delete all the provided typings :)